doc/nom_recipes.md

```diff
@@ -99,7 +101,7 @@ pub fn pinline_comment<'a, E: ParseError<&'a str>>(i: &'a str) -> IResult<&'a st
       take_until("*)"),
       tag("*)")
     )
-  )(i)
+  ).parse(i)
```

Comment:
Why use `parse` on all these instead of parser functions?

https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=3479d0fd619dd9886839429c6f9cc9a1

Reply:
I think it is required since #1631 has been merged.

Reply:
nom is moving towards using traits instead of function as the main parser interface. You can still use functions though if you prefer